### ISSUE TITLE
Sign of non-accuracy metrics

### ISSUE THREAD

**Issue by chreman** [maxpumperla/hyperas#115]
Sorry if I missed it in the documentation, if I compile the model with a different metric, e.g.
```python
    model.compile(optimizer={{choice(['rmsprop', 'adadelta'])}},
                  loss='mae',
                  metrics=['mae'])
```
and then call `score, mae = model.evaluate(...)` do I need to change the sign in 
`return {'loss': mae, 'status': STATUS_OK, 'model': model}`?
Intuitively I did so, because with accuracy higher=better, but with mean absolute error lower=better. If this is the case, a small update to the documentation would help removing this ambiguity. Thanks!

**Comment by vagarwal87** [comment]
yes, I read somewhere that hyperopt always attempts to minimize, thus metrics such as mae/mse should not be negated unlike accuracy.
